Website/src/rendering/src/components/SessionInformation.tsx

```diff
@@ -38,15 +51,53 @@ export type SessionInformationProps = ComponentProps & {
     Date: Field<string>;
     Image: ImageField;
     Speakers: Speaker[];
+    Rooms: Room[];
+    Day: Field<string>;
+    Timeslots: Timeslot[];
   };
 };
 
+function getTimeString(time: number, isEndTime: boolean) {
```

Comment:
I see this function in a couple of files. This is not the role of components to know how to format a time string. This should be moved to a utility file related to dates and time.